src/stores/datastore.js

```diff
@@ -1,35 +1,37 @@
+'use strict'
+
 const PREFIX_LENGTH = 8
 
 exports = module.exports
 
 exports.setUp = (basePath, blobStore, locks) => {
-  var store = blobStore(basePath + '/blocks')
+  const store = blobStore(basePath + '/blocks')
```

Comment:
It was here, in the version that was installed from npm, looks like I didn't fix it but the fix is not yet released to npm